revel/run.go

```diff
@@ -31,41 +32,92 @@ You can set a port as an optional third parameter.  For example:
     revel run github.com/revel/samples/chat prod 8080`,
 }
 
+// RunArgs is hold revel run parameters
+type RunArgs struct {
+	ImportPath string
+	Mode       string
+	Port       int
+}
+
 func init() {
 	cmdRun.Run = runApp
 }
 
-func runApp(args []string) {
-	if len(args) == 0 {
-		errorf("No import path given.\nRun 'revel help run' for usage.\n")
+func parseRunArgs(args []string) *RunArgs {
+	argsLength := len(args)
+	ia := RunArgs{
+		ImportPath: getImportPathCurrentDir(),
+		Mode:       DefaultRunMode,
+		Port:       revel.HTTPPort,
 	}
 
-	// Determine the run mode.
-	mode := DefaultRunMode
-	if len(args) >= 2 {
-		mode = args[1]
+	if argsLength == 3 {
```

Comment:
Why isn't there a comment block here providing possible argument values like the other cases?